From: Marco Martin <notmart@gmail.com>
Date: Mon, 19 May 2014 08:37:37 +0000
Subject: remove qt_filedialog_* api
X-Git-Url: http://quickgit.kde.org/?p=qtcurve.git&a=commitdiff&h=323402d8b99502300f65e909c61153cbd5d5d997
---
remove qt_filedialog_* api

it has been removed and doesn't build anymore with recent Qt5 stable
branch
---


--- qt5/style/qtcurve_api.cpp
+++ qt5/style/qtcurve_api.cpp
@@ -67,25 +67,6 @@
 
 #include <QDebug>
 
-extern QString (*qt_filedialog_existing_directory_hook)(
-    QWidget *parent, const QString &caption, const QString &dir,
-    QFileDialog::Options options);
-
-extern QString (*qt_filedialog_open_filename_hook)(
-    QWidget *parent, const QString &caption, const QString &dir,
-    const QString &filter, QString *selectedFilter,
-    QFileDialog::Options options);
-
-extern QStringList (*qt_filedialog_open_filenames_hook)(
-    QWidget * parent, const QString &caption, const QString &dir,
-    const QString &filter, QString *selectedFilter,
-    QFileDialog::Options options);
-
-extern QString (*qt_filedialog_save_filename_hook)(
-    QWidget *parent, const QString &caption, const QString &dir,
-    const QString &filter, QString *selectedFilter,
-    QFileDialog::Options options);
-
 namespace QtCurve {
 
 void
@@ -161,13 +142,6 @@
         if(!qtcIsFlatBgnd(opts.menuBgndAppearance) && 0 == opts.lighterPopupMenuBgnd)
             opts.lighterPopupMenuBgnd=1; // shade so that we dont have 3d-ish borders...
         opts.menuBgndAppearance = APPEARANCE_FLAT;
-    }
-
-    if (opts.useQtFileDialogApps.contains(appName)) {
-        qt_filedialog_existing_directory_hook = 0L;
-        qt_filedialog_open_filename_hook = 0L;
-        qt_filedialog_open_filenames_hook = 0L;
-        qt_filedialog_save_filename_hook = 0L;
     }
 
     QCommonStyle::polish(app);


